innovation.game.php

```diff
@@ -15729,6 +15730,7 @@ function stInterInteractionStep() {
                             if (!$artifact_found) {
                                 while (($top_card = self::getTopCardOnBoard($id, $returned_color)) !== null) {   
                                     // "Transfer all cards of the same color from the boards of all players with no top artifacts to your score pile."
+                                    $top_card = self::getCardInfo($top_card['id']);
```

Comment:
This bug doesn't actually happen when the card is in a publicly visible location (e.g. revealed or board). So let's remove this line so that we aren't making an unnecessary SQL query.